CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# Code owners groups assigned to this repository and a brief description of their areas:
+# @cilium/ci-structure       Continuous integration, testing
+# @cilium/contributing       Developer documentation & tools
+# @cilium/github-sec         GitHub security (handling of secrets, consequences of pull_request_target, etc.)
+# @cilium/hubble             All related to Hubble (client and server)
+# @cilium/vendor             Vendoring, dependency management
+
+# The following filepaths should be sorted so that more specific paths occur
+# after the less specific paths, otherwise the ownership for the specific paths
+# is not properly picked up in Github.
+* @cilium/hubble
```

Comment:
Not sure what default subteam to use here. A few members of Hubble team historically have had the most knowledge of this component, but it is also used by clustermesh-apiserver (for which I have not found a team).

The important thing is that `/.github/workflows/` is owned by `@cilium/github-sec`.